chainer/function_hooks/debug_print.py

```diff
@@ -50,13 +49,14 @@ class PrintHook(function_hook.FunctionHook):
     name = 'PrintHook'
 
     def __init__(self, sep='', end='\n', file=sys.stdout, flush=True):
+        # TODO(niboshi): deprecate sep
```

Comment:
Let's remove this line as https://github.com/chainer/chainer/pull/4471 got merged.